concrete/blocks/form/controller.php

```diff
@@ -397,8 +397,12 @@ public function action_submit_form($bID = false)
             if (intval($row['required']) == 1) {
                 $notCompleted = 0;
                 if ($row['inputType'] == 'email') {
-                    if (!Core::make('helper/validation/strings')->email($_POST['Question' . $row['msqID']])) {
-                        $errors['emails'] = t('You must enter a valid email address.');
+                    if (!isset($emailValidator)) {
+                        $emailValidator = $this->app->make(EmailValidator::class);
+                    }
+                    $e = $this->app->make('error');
+                    if (!$emailValidator->isValid($_POST['Question' . $row['msqID']], $e)) {
```

Comment:
Do we have to use `$_POST` directly here? I'm fairly certain we have access to `$this->request` here

Reply:
I just kept the existing code, but for sure I can update it.

Reply:
I think it's worth not using the superglobals, but the rest of the things are just requests